FrontEnd/src/components/ProfilePage/FormComponents/FormFields/ImageField.jsx

```diff
@@ -2,62 +2,83 @@ import classNames from 'classnames';
 import preventEnterSubmit from '../../../../utils/preventEnterSubmit';
 import css from './ImageField.module.css';
 
-const ImageField = (props) => {
+const ImageField = ({
+  name,
+  label,
+  accept,
+  inputType = 'text',
+  value,
+  updateHandler,
+  onDeleteImage,
+  error,
+}) => {
   const backgroundImage = {
-    background: `url(${props.value}) lightgray 50% / cover no-repeat`,
+    background: `url(${value}) lightgray 50% / cover no-repeat`,
   };
 
-  return (
-    <div
-      className={classNames(css['fields__column'], {
-        [css['fields__column--logo']]: props.name === 'logo_image',
-      })}
+  const renderInput = () => (
+    <input
+      accept={accept}
+      id={name}
+      type={inputType}
+      className={css['upload-file__input']}
+      name={name}
+      onChange={updateHandler}
+      onKeyDown={preventEnterSubmit}
```

Comment:
I think prevent default should be added to the submit button. Does the current implementation work correctly with the tab navigation?

Reply:
Yes, current implementation works well for tab navigation. 
Prevent Enter submit was added to all input fields to prevent form being submitted with Enter key, when accidentally pressed in any input field. Because it is unexpected and unwanted button behavior for the user.